cmd/readme_test/readme.go

```diff
@@ -0,0 +1,53 @@
+package main
```

Comment:
Is there a reason why you didnt modify `cmd/test/main.go` to reflect the readme ?
I dont see a need to have 2 different sample code both are for the same purpose.